style/predict/servable/base.py

```diff
@@ -29,11 +33,12 @@ def __init__(self, model):
         self.model = model
 
     def export(self, path):
-        pass
+        with open(path, "wb") as f:
+            dill.dump(self.model, f)
 
     def run_inference(self, texts):
         pass
 
     @classmethod
     def load(cls, path):
-        pass
+        dill.load(path)
```

Comment:
this is a `classmethod` and in most cases, it returns the new object. so here it should be:
```suggestion
        return cls(model=dill.load(path))
```
does it make sense?

Reply:
Yes. it does.